modules/dxp/apps/search-experiences/search-experiences-service/src/test/java/com/liferay/search/experiences/internal/suggestions/spi/SetUpSearcherMockData.java

```diff
@@ -0,0 +1,44 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2025 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.search.experiences.internal.suggestions.spi;
+
+import com.liferay.portal.kernel.model.ClassName;
+import com.liferay.portal.search.hits.SearchHit;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * @author Rodrigo Guedes de Souza
+ */
+public class SetUpSearcherMockData {
```

Comment:
Moving this Helper to portal-search-test-util should make it so we don't need a duplicate class.